test/unit/unit-entry.js

```diff
@@ -107,8 +107,7 @@ const TEST_GLOBAL = {
   DownloadsViewUI: {DownloadElementShell},
   Services: {
     locale: {
-      getAppLocaleAsLangTag() { return "en-US"; },
-      getAppLocalesAsLangTags() {},
+      get appLocaleAsLangTag() { return "en-US"; },
```

Comment:
Hmm.. I guess as a getter, a missing `appLocalesAsLangTags` returns `undefined` which is what the original method was returning (but the method needed to be defined as trying to call `undefined` as a function would throw).. I guess this is fine 😉